contents/blog/product-engineer-tools.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2022-12-14
+title: The best tools for product engineers
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+categories: ["Startups", "Guides"]
+---
+
+Building tools for product engineers is a big trend right now. Every week brings a flashy launch of a new tool intersecting product and engineering. This is because building a great product users value is a proven way to succeed as a business. Helping product engineers make this happen is valuable.
+
+Tools for product engineers differ from both the tools for software engineers and product managers. The engineering tools focus on speed and automation rather than the traditional customization and stability. The product tools are streamlined for technical users, rather than being an all-encompassing “business tool.”
+
+This post will go over the tools built and used to help product engineers accomplish their work of building great products for users, from the beginning stages of talking to users all the way to maintaining ownership over issues and bugs in production.
```

Comment:
Can you get this intro down to two paras? Best to keep intros on these types of articles as tight and actionable as possible. If one para is possible, even better.